Symfony/CS/Tests/Fixer/Contrib/LongArraySyntaxFixerTest.php

```diff
@@ -19,6 +19,25 @@
 class LongArraySyntaxFixerTest extends AbstractFixerTestBase
 {
     /**
+     * @requires PHP 5.4
+     * @dataProvider provideSkipExamples
+     */
+    public function testSkip($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideSkipExamples()
+    {
+        return array(
+            array('<?php $x[2] = 1;'),
+            array('<?php $x["a"] = 1;'),
+            array('<?php $x[] = 1;'),
+            array('<?php $x[ ] = 1;'),
```

Comment:
To be fixed? don't think so, but I'm not sure, for the tests to pass yes it is needed.
We are taking another route now so I didn't change any other short array test code until we figured out what to do, more discussing is in this PR:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1563#discussion_r46339790